tests/test_oml/test_datasets/test_list_dataest.py

```diff
@@ -0,0 +1,19 @@
+import torch
```

Comment:
There are no tests for a case with bounding boxes. Please add a test according to the new recommendation of `test_dataset_iter` test.

You can find csv with bounding boxes in `MOCK_DATASET_PATH / "df_with_bboxes.csv". Or you can create random bboxes.

You also can parameterize `test_dataset_iter` test with bounding boxes. It's up to you.